usr/lib/thingy/thingy.py

```diff
@@ -189,6 +193,38 @@ def on_menu_quit(self, widget):
     def on_app_changed(self, widget):
         self.load_documents()
 
+    def on_search_changed(self, widget):
+        search_text = self.app_search.get_text().lower().strip()
+        self.filter_documents(filter=search_text)
+    
+    @_async
+    def filter_documents(self, filter=None):
+        
+        self.documents = []
+        self.clear_flowbox()
+
+        if(len(filter) == 0):
+            return
+
+        #Favorites
+        items = self.favorites_manager.get_favorites(None)
+        for item in items:
+            if filter in item.get_display_name():
```

Comment:
There is no `item.get_display_name()` (XApp.FavoriteInfo), use `item.display_name`.

https://lazka.github.io/pgi-docs/#XApp-1.0/classes/FavoriteInfo.html#XApp.FavoriteInfo